service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/DocmosisCaseManagementOrder.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.fpl.model.docmosis;
+
+import lombok.Builder;
+import lombok.Getter;
+import uk.gov.hmcts.reform.fpl.model.common.Schedule;
+
+import java.util.List;
+
+@Getter
+public class DocmosisCaseManagementOrder extends AbstractDocmosisOrder {
+    private final List<DocmosisRepresentative> representatives;
+    private final Schedule schedule;
+    private final boolean scheduleProvided;
+
+    private final List<DocmosisRecital> recitals;
+
+    //not needed can manage in doc
+    private final boolean recitalsProvided;
+
+    //not needed can manage in doc
+    private final int numberOfChildren;
```

Comment:
is this going to be done in this PR or in a future date?

Reply:
doing now!